sandbox/tests/test scenes/many light sampling/20x20_white_point_lights.appleseed

```diff
@@ -4899,7 +4899,52 @@
     </output>
     <configurations>
         <configuration name="final" base="base_final">
+            <parameter name="lighting_engine" value="pt" />
```

Comment:
to be honest, I am not sure where did this come from as i didn't go directly into the file. Maybe I saved it by accident from the studio.